esphome/components/mopeka_pro_check/mopeka_pro_check.cpp

```diff
@@ -50,9 +50,9 @@ bool MopekaProCheck::parse_device(const esp32_ble_tracker::ESPBTDevice &device)
     return false;
   }
 
-  // Now parse the data - See Datasheet for definition
 
```

Comment:
Add back original comment
```suggestion
// Now parse the data - See Datasheet for definition

```